Data-Science/Kubeflow/MNIST-Digits-Recognition/mnist_katib_tf_kserve_example.ipynb

```diff
@@ -21,6 +21,9 @@
     "# # You need to generate secret to have ability read datasource files from S3 bucket by Spark application (Airflow DAG)\n",
     "# # Copy template \"object_store_secret.yaml.tpl\" from shared/ezua-tutorials/current-release/Data-Analytics/Spark directory to e.g. user folder\n",
     "\n",
+    "# # Customers need to install following images manually in order to run the example.\n",
```

Comment:
Rather than install, better statement would be "user have to pull the images manually and made available in local repo for clusters in airgap network"

Reply:
Yeah, updated the comment.